configuration-ghc-810.nix

```diff
@@ -0,0 +1,58 @@
+{ pkgs, inputs }:
+
+let
+  disabledPlugins = [
+    "hls-brittany-plugin"
```

Comment:
Theses plugins are broken with GHC 8.10?

Reply:
Copy/paste, I don't know. Can iterate on this too, on a separate task

Reply:
Fixed